### DISCUSSION TITLE
Cannot draft a release anymore

### DISCUSSION THREAD

**Discussion by Shinigami92** [community/community#5672] [Other Feature Feedback, Questions, & Ideas] [answered]
I try to follow this documentation: https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#creating-a-release

But I cannot find the `Draft a new release` button anymore. Some weeks ago it worked, but now it's gone :shrug: 

Trying to create a tag via the following option also doesn't work :slightly_frowning_face: 
<img src="https://user-images.githubusercontent.com/7195563/133603742-19f2c148-4205-455d-b6cd-8cf7930c7527.jpg" width="300px"/>

So currently I have NO options to create a release or tag via GitHub itself and I need to create a tag manually via terminal and then I can create a release with this tag.
This worked before and now it's really an annoying process :confused:

**Comment by MylesBorins** [comment, marked as answer]
@Shinigami92 here's a video of me going through the whole flow end-to-end. Please let me know if there is anything different on your end or anything in the docs that specifically doesn't make sense.

https://user-images.githubusercontent.com/498775/139342303-7f8b41fa-0c13-4041-a09f-14fc774dbdf9.mp4

**Reply by Shinigami92** [reply]
Thank you
Seems that it got updated in the meantime
Before it was not possible (or unclear?)

In the past before the changes, it was possible to just say: name the tag exactly like the version number or release title or somewhat, now you need to explicitly insert the version and then click on the `Create new tag`

Beside that, it's not directly clear to me (as a user) what this greyed out `on publish` means.
The tag will be created when the draft gets un-drafted?

**Comment by jkbahian** [comment]
thanks man
